convert.py

```diff
@@ -0,0 +1,52 @@
+import binascii
```

Comment:
no, I meant like this: https://github.com/tomato42/ecpp-verifier/blob/master/LICENSE#L460